packages/back-end/src/models/InformationSchemaTablesModel.ts

```diff
@@ -62,15 +65,92 @@ const toInterface = (
   doc: InformationSchemaTablesDocument
 ): InformationSchemaTablesInterface => omit(doc.toJSON(), ["__v", "_id"]);
 
-export async function createInformationSchemaTables(
-  tables: InformationSchemaTablesInterface[]
-): Promise<InformationSchemaTablesInterface[]> {
-  //TODO: Remove this check and orgs usingFileConfig to create informationSchemas
+export async function createInformationSchemaTable(
```

Comment:
This change came about because we are now only creating the table records in a `just-in-time` manner, rather than creating them all at once when a datasource is created.